libs/langchain/langchain/chat_models/base.py

```diff
@@ -55,19 +68,35 @@ def init_chat_model(
                 - gemini... -> google_vertexai
                 - command... -> cohere
                 - accounts/fireworks... -> fireworks
+        configurable: Whether model parameters are configurable. Defaults to True if
+            model is None or config_prefix is non-null or configure_any is True.
+        config_prefix: If config_prefix is a non-empty string then model will be
+            configurable at runtime via the
+            ``config["configurable"]["{config_prefix}_model"]`` and
+            ``config["configurable"]["{config_prefix}_model_provider"]`` keys. If
+            config_prefix is an empty string then model will be configurable via
+            ``config["configurable"]["model"]`` and
+            ``config["configurable"]["model_provider"]`` keys.
+        configure_any: If True then any parameter can be passed to model initializer as
```

Comment:
config_any=True seems like an easy way to steal someones api keys or potentially do a man in the middle attack by changing the api url

Could we add an option to secify specific field names?